src/server.js

```diff
@@ -58,7 +59,7 @@ app.get('/api/actors', actorController.getActors, (req, res) => {
 
 // add a new actor to the db, return the actor object
 app.post('/api/actors', actorController.newActor, (req, res) => {
-  return res.status(201).json(res.locals.actor);
+  return res.status(200).json(res.locals.actor);
```

Comment:
Don't we want 201 (created) instead of 200?  
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201  
Not a big deal though.